packages/interop-tests/src/__test__/interop.test.ts

```diff
@@ -148,25 +192,79 @@ it('server + browser wallet interoperability test', async () => {
    */
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  const postFundAPromise = fromEvent<SingleChannelOutput>(serverWallet as any, 'channelUpdated')
+  const postFundA = await fromEvent<SingleChannelOutput>(serverWallet as any, 'channelUpdated')
     .pipe(take(3))
     .toPromise();
 
+  await browserWallet.pushMessage(serverMessageToBrowserMessage(postFundA), 'dummyDomain');
+
+  await objectiveSuccededPromise;
+});
+
+it('browser wallet creates channel + cooperates with server wallet to fund channel', async () => {
+  const createChannelParams: CreateChannelParams = {
```

Comment:
Could (should?) we use the same parameters here?

In an ideal world, we could execute the same test with roles reversed. Even better, we could execute the same test for many combinations of actors:
- `[server, server]`
- `[server, browser]`
- `[browser, server]`
- `[browser, browser]`

We are not in that world right now, because the APIs differ, correct?

Reply:
It would be nice to create a test suite that tests all of these combinations. At the moment:
- create channel API differs between server and browser wallet.
- message passing between server and browser wallets needs to be cleaned up in the interop test.